No such format 'gnu'

Build fails on MacOS:
```
% mvn clean verify -Dmaven.test.skip
....
[INFO] [main] + unzip -qq ../org.eclipse.oomph.setup.installer.product.with-jre-win32.win32.aarch64.zip
[INFO] [main] + tar --format=gnu -cf ../org.eclipse.oomph.setup.installer.product.with-jre-win32.win32.aarch64.tar artifacts.xml configuration eclipse-inst.exe eclipse-inst.ini eclipse-instc.exe features p2 plugins readme
[INFO] [main] Can't use format gnu: No such format 'gnu'
[INFO] [main] Usage:
[INFO] [main]   List:    tar -tf <archive-filename>
[INFO] [main]   Extract: tar -xf <archive-filename>
[INFO] [main]   Create:  tar -cf <archive-filename> [filenames...]
[INFO] [main]   Help:    tar --help
[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
    at org.apache.commons.exec.DefaultExecutor.executeInternal (DefaultExecutor.java:404)
    at org.apache.commons.exec.DefaultExecutor.execute (DefaultExecutor.java:166)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:982)
    at org.codehaus.mojo.exec.ExecMojo.executeCommandLine (ExecMojo.java:929)
    at org.codehaus.mojo.exec.ExecMojo.execute (ExecMojo.java:452)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:903)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:280)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:203)
    at jdk.internal.reflect.DirectMethodHandleAccessor.invoke (DirectMethodHandleAccessor.java:103)
    at java.lang.reflect.Method.invoke (Method.java:580)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
....
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:3.0.0:exec (repackage-products) on project org.eclipse.oomph.setup.installer.product: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]

```

--- a/products/org.eclipse.oomph.setup.installer.product/repackage-products.sh
+++ b/products/org.eclipse.oomph.setup.installer.product/repackage-products.sh
@@ -35,7 +35,7 @@ for i in $(ls); do
       mkdir -p tmp
       cd tmp
       unzip -qq ../$i
-      tar --format=gnu -cf ../${i%.zip}.tar *
+      tar --format=pax -cf ../${i%.zip}.tar *
       cd ..
       rm -rf tmp
       product=${i%.zip}.tar

